server/server_const_windows_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build windows
```

Comment:
Why the changed build tag format? Do we need both?

Reply:
Yes, it's auto-generated by `go fmt` on 1.17 and `go vet` complains if you try to delete the old form. Notes start [here](https://golang.org/doc/go1.17#gofmt).